swarm/api/http/templates.go

```diff
@@ -69,3 +69,149 @@ var htmlListTemplate = template.Must(template.New("html-list").Funcs(template.Fu
+        </div>
+        <div class="page-title">
+          <h1>Welcome to Swarm</h1>
+        </div>
+        <div class="header-right">
+          <div id="timestamp">{{.Timestamp}}</div>
```

Comment:
This variable is not being bound in the template, we should either pass in a variable for `Timestamp` or just drop this header (does the status page really need to show the time?).